### PR TITLE
Carbon2 configuration option - include field in metric name

### PR DESCRIPTION
**Pull request by pmalek-sumo**
### Required for all PRs:

- [x] Signed [CLA](https://influxdata.com/community/cla/).
- [x] Associated README.md updated.
- [x] Has appropriate unit tests.

### CODE REVIEW

File: plugins/serializers/carbon2/README.md [hunk 1]
```diff
@@ -14,20 +14,51 @@ The `carbon2` serializer translates the Telegraf metric format to the [Carbon2 f
   ## more about them here:
   ## https://github.com/influxdata/telegraf/blob/master/docs/DATA_FORMATS_OUTPUT.md
   data_format = "carbon2"
+
+  ## Optionally configure metrics format, whether to merge metric name and field name.
+  ## Possible options:
+  ## * "field_separate"
+  ## * "metric_includes_field"
+  ## * "" - defaults to "field_separate"
+  carbon2_format = ""
```
Comment (ssoroka) [inline plugins/serializers/carbon2/README.md:23]:
```suggestion
  # carbon2_format = "field_separate"
```

**Review by ssoroka — APPROVED** [review]
Looks good. Just have a minor suggestion for the readme.

File: plugins/serializers/carbon2/README.md (hunk 1 above)
Comment (pmalek-sumo) [reply plugins/serializers/carbon2/README.md]: Applied your suggestion, thanks!

**Comment by pmalek-sumo** [comment]
It seems all tests have passed and this is ready to be merged 👍🏻

**Review by ssoroka — APPROVED** [review]
looks great

**Pull request merged**